numpy/core/src/multiarray/number.c

```diff
@@ -173,9 +181,24 @@ PyArray_GetNumericOps(void)
 
  fail:
     Py_DECREF(dict);
+    PyErr_SetString(PyExc_RuntimeError,
+        "one or more array ops not set");
```

Comment:
Not sure there's much point in this - an exception is already thrown here anyway

Reply:
reverted (removed)